task12.1/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task12._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите натуральное число m (5 < m < 20)");
+            int m;
+            if (!TryInputNumber(out m))
+            {
+                Console.ReadKey();
+                return;
+            }
+
+            Console.WriteLine("Введите натуральное число n (5 < n < 20)");
+            int n;
+            if (!TryInputNumber(out n))
+            {
+                Console.ReadKey();
+                return;
+            }
+
+            if (m < 5 || m > 20 || n < 5 || n > 20)
+            {
+                Console.WriteLine("Числа не удовлетворяют неравенству.");
+                Console.ReadKey();
+                return;
+            }
+
+            var matrix = new int[m, n];
+
+            var rnd = new Random();
+
+            for (int i = 0; i < matrix.GetLength(0); i++)
+                for (int j = 0; j < matrix.GetLength(1); j++)
+                    matrix[i, j] = rnd.Next(100);
+            PrintMatrix(matrix);
+
+            Console.WriteLine();
+
+            var (columnIndex1, rowIndex1, columnIndex2, rowIndex2) = GetIndexOfColumnWithMinimalLastElement(matrix);
+
+            Console.WriteLine($"Элемент с индексами({columnIndex1}, {rowIndex1}) > элемента с индексами({columnIndex2}, {rowIndex2})");
```

Comment:
Когда индексируют элементы матрицы, сначала пишут индекс строки, а затем столбца
```suggestion
            Console.WriteLine($"Элемент с индексами({rowIndex1}, {columnIndex1}) > элемента с индексами({rowIndex2}, {columnIndex2})");
```